app/src/ui/repository.tsx

```diff
@@ -78,6 +78,17 @@ export class RepositoryView extends React.Component<
     }
   }
 
+  public shouldComponentUpdate(
+    nextProps: IRepositoryViewProps,
+    nextState: IRepositoryViewState
+  ) {
+    const shouldUpdate = nextProps !== this.props || nextState !== this.state
+
+    console.dir(diffObj(this.props, nextProps))
```

Comment:
Logging out props in an effort to see which props are changing since I cannot figure out why renderers are rendering all over the place.